Packages.props

```diff
@@ -18,9 +18,9 @@
 		<PackageReference Update="Microsoft.Azure.Management.Sql" Version="1.41.0-preview" />
 
 		<PackageReference Update="Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider" Version="1.1.1" />
-		<PackageReference Update="Microsoft.Data.SqlClient" Version="2.1.3"/>
+		<PackageReference Update="Microsoft.Data.SqlClient" Version="3.0.0"/>
 		<PackageReference Update="Microsoft.SqlServer.SqlManagementObjects" Version="161.46367.54" />
-		<PackageReference Update="Microsoft.SqlServer.DACFx" Version="150.5290.2-preview" GeneratePathProperty="true" />
+		<PackageReference Update="Microsoft.SqlServer.DACFx" Version="160.5317.5-preview" GeneratePathProperty="true" />
 		<PackageReference Update="Microsoft.Azure.Kusto.Data" Version="9.0.4" />
 		<PackageReference Update="Microsoft.Azure.Kusto.Language" Version="9.0.4"/>
 		<PackageReference Update="Microsoft.SqlServer.Assessment"  Version="[1.0.305]" />
```

Comment:
@aasimkhan30 , I see the version is mismatched with two version for <PackageReference Update="Microsoft.SqlServer.Management.SqlParser" Version="160.21267.55" />, can you confirm if I can update it to either 160.21267.51 or 160.21292.55. I don't see v160.21267.55 in [nuget.org](https://www.nuget.org/packages/Microsoft.SqlServer.Management.SqlParser/). please correct me if i'm missing any info here, Thanks :)

Reply:
@ssreerama version 160.21292.55 should be the version. Can you please make the change here?